foyer-memory/benches/bench_hit_ratio.rs

```diff
@@ -224,6 +227,37 @@ fn bench_zipf_hit() {
     }
 }
 
+fn read_twitter_trace(path: &str, limit: usize) -> Vec<String> {
+    let file = std::fs::File::open(path).unwrap();
+    let mut reader = Reader::from_reader(file);
+    let mut keys = Vec::new();
+    for result in reader.records() {
+        let record = result.unwrap();
+        let key = record.get(1).unwrap().to_string();
+        keys.push(key);
+        if keys.len() >= limit {
+            break;
+        }
+    }
+    keys
+}
+
 fn main() {
+    // Try to read the csv file path by environment variable.
+    let path = std::env::var("TWITTER_TRACE_PATH").ok();
+    if let Some(path) = path {
+        // Limit the number of keys to read.
+        // MAX means read all keys, which may take a really long time.
```

Comment:
Add an interactive command to download the dataset?

Reply:
Download this is a little bit complex.

Reply:
> Download this is a little bit complex.

Hi @xiaguan , where can I find the dataset? Let me handle the download part.

Reply:
https://github.com/twitter/cache-trace, I think I used miss ratio related cluster's data.
I use SNIA to download. In this way, I could just download part of it(6GB).

Reply:
https://github.com/twitter/cache-trace this one?

Reply:
Yep, it is really big. 🤣